ragna/source_storages/__init__.py

```diff
@@ -2,11 +2,15 @@
     "Chroma",
     "LanceDB",
     "RagnaDemoSourceStorage",
+    "MiniLML6v2",
+    "VectorDatabaseSourceStorage",
 ]
 
+from ._vector_database import VectorDatabaseSourceStorage
```

Comment:
Why did this become public? This is an internal helper class that we can probably remove entirely after the chunking PR is landed as well. Please make it private again.